app/src/gui/component_registry/bundle_components.ts

```diff
@@ -293,6 +294,15 @@ registerField(
   FAIMSTextField
 );
 
+registerField(
+  'faims-custom',
+  'NumberField',
+  'Number Input Field',
+  'Allows users to input numeric values',
+  'Text',
```

Comment:
Should the 'category' be something else maybe 'Number'?

Reply:
Maybe check with Steve what the intent of this 'category' is

Reply:
I think it was to group the fields into similar types but not used at present.  More relevant in Designer where this info is not available (it has it's own list of fields).